docs/docs.polserver.com/pol100/objref.xml

```diff
@@ -309,6 +309,8 @@
 <member mname="luck_mod" type="Integer" access="r/w">Modifier for property, added to base property.</member>
 <member mname="swing_speed_increase_mod" type="Integer" access="r/w">Modifier for property, added to base property.</member>
 <member mname="buffs" type="dictionary" access="r/o">Returns the current active buffs added via addBuff as dictionary. Key is the iconid and value a struct struct{name_cliloc, desc_cliloc, end_time, name_args, desc_args}.</member>
+<member mname="position_changed_at" type="Integer" access="r/o">PolClock when character last changed position for any reason</member>
```

Comment:
Instead PolClock I would suggest to mention the MilliSecondClock which you use to store the value. And you return a Double and not a Integer.

Reply:
Double is correct otherwise it will overflow after a few days.
The others return in 10ms resolution, not really usable I guess, but it is what it is. :)